src/main/scala/scorex/core/api/http/PeersApiRoute.scala

```diff
@@ -55,7 +57,8 @@ case class PeersApiRoute(peerManager: ActorRef,
       case Some(addressAndPort) =>
         val host = InetAddress.getByName(addressAndPort.group(1))
         val port = addressAndPort.group(2).toInt
-        networkController ! ConnectTo(new InetSocketAddress(host, port))
+        val peerInfo = PeerInfo(timeProvider.time(), new InetSocketAddress(host, port), None, None, Seq())
```

Comment:
Let's allow to provide `nodeName` (not sure about connectionType) via API instead of setting it always to `None`.